src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -234,7 +239,8 @@ public ResponseEntity findUsersByOrganisation(@ApiParam(hidden = true) @OrgId St
             value = "/users/accountId",
             produces = MediaType.APPLICATION_JSON_UTF8_VALUE
     )
-    @Secured({"pui-finance-manager", "pui-user-manager", "pui-organisation-manager", "pui-case-manager", "caseworker-publiclaw-courtadmin"})
+    //@Secured({"pui-finance-manager", "pui-user-manager", "pui-organisation-manager", "pui-case-manager", "caseworker-publiclaw-courtadmin"})
+    @Secured({"dummy-role"})
```

Comment:
Commented out the roles instead of deleting the line so it is easier for us to revert back in future. Added "dummy-role" as the only role able to access the endpoint.